### DISCUSSION TITLE
support for CONECT in parsePDB and writePDB

### DISCUSSION THREAD

**Discussion by grinner** [prody/ProDy#1884] [Archived Issues]
I'd like to add support for CONECT records in pdb files.   I figure I need to
1. Add a Bond and/or a BondGroup class
2. possibly extend AtomGroup to have a BondGroup property

I have an application where I need to merge pdb files the have both ATOM and CONECT records

I am going to do something in my own fork, but any suggestions as to the interface I implement would be welcome so I can merge the changes upstream.

While setBonds exists in AtomGroup, it doesn't appear to support Hydrogen bonding, so any guidance on conformant style would be welcome.

**Comment by cihankayacihan** [comment]
Thanks for your input. Can you send this over cihank@pitt.edu so that we can discuss more and find a better way to implement this. Thanks.

**Comment by w-ahern** [comment]
I see this is closed as completed, but while now parsePDB can parse CONECT records, writePDB cannot write them.  For me this means that ProDy cannot be used to manipulate structures which contain heteroatom records.

**Comment by jamesmkrieger** [comment]
Thanks for the comment. I absolutely agree we still need that
